tests/tests/test_preauth.py

```diff
@@ -239,7 +237,7 @@ def substitute_id_data(device, id_data_dict):
     def restart(device):
         """Restart the mender service."""
 
-        device.run("systemctl restart %s.service" % device.get_client_service_name())
+        device.run("systemctl restart mender-updated")
```

Comment:
We shuuure this should not be `auth`?

Reply:
This test actually passes, so it needs to be right.

However I share with you the hunch that it _should_ be `mender-authd`. Which brings me to the realization that we actually don't need to restart the service at all: `mender-auth` will pick up the updated id script once it is modified on disk. I'll remove the restart for clarity.